### PR TITLE
Description for new http interface added

### PR DESCRIPTION
**Pull request by scarface-4711**
**Description:**

Added a very short description and configuration example for my new denon_http interface

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#4580

### CODE REVIEW

File: source/_components/media_player.denon.markdown [hunk 1]
```diff
@@ -22,15 +22,27 @@ Supported devices:
...
 
+Telnet interface
 ```yaml
 # Example configuration.yaml entry
 media_player:
   - platform: denon
     host: IP_ADDRESS
 ```
+
+http interface
```
Comment (Landrash) [inline source/_components/media_player.denon.markdown]: HTTP and not http.

File: source/_components/media_player.denon.markdown [hunk 2]
```diff
@@ -43,3 +55,5 @@ A few notes:
 - To be able to wake up the receiver, activate the "remote" setting in the receiver's settings.
 - Play and pause are supported, toggling is not possible.
 - Seeking cannot be implemented as the UI sends absolute positions. Only seeking via simulated button presses is possible.
+- Additional option the control Denon receivers with a builtin web server is using the http interface
```
Comment (Landrash) [inline source/_components/media_player.denon.markdown]: Should probably be reworded to 
> The HTTP interface supports some additional functionalities like album covers and supports more than one simultaneous connection.

File: source/_components/media_player.denon.markdown [hunk 3]
```diff
@@ -22,15 +22,27 @@ Supported devices:
 - Denon RCD-N8 (untested)
 - Denon RCD-N9 (partial support)
 - Denon AVR receivers with Integrated Network support (partial support)
+- Denon AVR-X4100W (via http interface)
```
Comment (Landrash) [inline source/_components/media_player.denon.markdown]: Should probably be HTTP instead of http.

**Comment by scarface-4711** [comment]
Did some repackaging and renamed my library.
@Landrash: included your feedback

**Pull request merged**